frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -41,11 +41,21 @@ export function MissionHeader({ mission }: MissionHeaderProps) {
         remainingTime = 'Not available'
     }
 
+    var showControlButtons = true
+    if (
+        mission.missionStatus === MissionStatus.Successful ||
+        mission.missionStatus === MissionStatus.Aborted ||
+        mission.missionStatus === MissionStatus.Cancelled ||
+        mission.missionStatus === MissionStatus.Failed
+    ) {
+        showControlButtons = false
+    }
```

Comment:
```suggestion
    var showControlButtons = false
    if (
        mission.missionStatus === MissionStatus.Ongoing ||
        mission.missionStatus === MissionStatus.Paused
    ) {
        showControlButtons = true
    }
```

Suggesting reducing the number of statuses to depend on